File: 0283-move-zeroes/0283-move-zeroes.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        int anc=0;
        for(int i=0;i<nums.size();i++){
            if(nums[i]){
                nums[anc++]=nums[i];
            } 
        }
        for(anc;anc<nums.size();anc++) nums[anc]=0;
    }
};

File: 0283-move-zeroes/0283-move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        anc = 0 
        for val in nums:
            if val: 
                nums[anc]=val 
                anc+=1 
        
        for i in range(anc,len(nums)): 
            nums[i]=0
        
        